cvmfs/cvmfs_config

```diff
@@ -254,13 +257,14 @@ cvmfs_chksetup() {
     fi
   fi
 
+  AUTOMASTER=$(find /etc -name "auto.master" 2>/dev/null | head -1)
```

Comment:
Sure. The path is `/etc/autofs/auto.master`. However, using `find` is more flexible, I think. Would it be an option to run the `find` during installation and save the found path in some text file? Otherwise, of course, simply hardcoding the path is simple enough as well.